packages/web-app-files/src/views/spaces/DriveRedirect.vue

```diff
@@ -53,21 +55,25 @@ export default defineComponent({
       return store.getters.homeFolder
     })
 
-    const { params, query } = createFileRouteOptions(unref(personalSpace), {
-      path: unref(itemPath)
-    })
-
-    router
-      .replace({
-        ...unref(route),
-        params: {
-          ...unref(route).params,
-          ...params
-        },
-        query
+    if (unref(useCapabilitySpacesEnabled()) && !unref(personalSpace)) {
```

Comment:
Same here, you don't need the capability check. The check for absence of a personal space is sufficient.
Although strictly looking at it, the redirect to the project space overview would require the project spaces capability... 🤔 I'd be fine with that implicit behaviour, because we know that a) it won't be an issue in oc10 environments and b) oc10 support will vanish at some (far away?) point in the future.